installer/main.sh

```diff
@@ -102,7 +102,7 @@ cd dns
 . setup.sh $DNS_DOMAIN_NAME
 cd ..
 
-if [[ "$INSTALL_MINIO" ]]; then
```

Comment:
`= true` would be enough here based on this
https://stackoverflow.com/questions/2953646/how-to-declare-and-use-boolean-variables-in-shell-script
but this solution also works, so lets merge it